src/card.css

```diff
@@ -0,0 +1,91 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
```

Comment:
on file with name card should be styles for the card. You can move these styles to a style file (or you can create main file)